config/secrets.yml.example

```diff
@@ -32,10 +33,12 @@ development:
 
 test:
   secret_key_base: 56792feef405a59b18ea7db57b4777e855103882b926413d4afdfb8c0ea8aa86ea6649da4e729c5f5ae324c0ab9338f789174cf48c544173bc18fdc3b14262e4
+  # time_zone: ""
   <<: *maps
 
 staging:
   # secret_key_base: ""
+  # time_zone: ""
```

Comment:
Note that the installer expects the `secret_key_base` to be followed by the `server_name`, so adding this key here might break that part :thinking:.

Reply:
Good catch! I moved the `time_zone` key after `server_name`, so as not to break the task to replace the `secret_key_base`.